support-services/src/main/scala/com/gu/support/promotions/dynamo/DynamoService.scala

```diff
@@ -33,7 +31,7 @@ object DynamoService {
     new InstanceProfileCredentialsProvider(false)
   )
 
-  def forTable[T](table: String)(implicit e: ExecutionContext, decoder: Decoder[T]): DynamoService[T] = {
+  def forTable[T](table: String)(implicit decoder: Decoder[T]): DynamoService[T] = {
```

Comment:
I just did some tidy up on this file anyway, I was going to put the new stuff here it but I realised none of it was actually related other than it also hitting dynamo

Reply:
Do we not need an execution context here?

Reply:
no, it compiles without, and all of the promo code stuff is using the blocking client anyway (probably shouldn't be but it hasn't caused an incident so far)